CONTRIBUTING.rst

```diff
@@ -0,0 +1,109 @@
+======================
+Contributing to DWI_ML
+======================
+
+Welcome to DWI_ML ! We're excited you're here and want to contribute.
+
+This article documents how to contribute improvements to DWI_ML.
+
+The development process
+=======================
+
+When writing code, please pay attention to the following aspects:
+
+Coverage
+--------
+
+We use `pytest`_ to write the tests of the code.
+
```

Comment:
Do we test only script, or all code functions? Do we test classes? Do we test classes that are already tested in a script?  Should there be more details here? Up to you, I am not really used to this.

Reply:
I would leave it as it is. No hard requirements. If people are willing to write them because it helps them (and possibly others), and have some time, then let them write them to the detail they are willing to.

Ideally, we should write tests for both. But our time limitations and other constraints may prevent us from getting to that point. I am fine with that.

If we write one every once in a while, at least we know the testing tool the repository uses.

> Do we test classes that are already tested in a script?

A script as we understand them in the context of `Scilpy`, `VITALabAI` or this repository is not really a test, and does not test the classes used either. A test would ideally compare the result of some call against an baseline that is known to be correct.